zulipterminal/ui_tools/boxes.py

```diff
@@ -1361,6 +1361,30 @@ def main_view(self) -> List[Any]:
         ]
         return [part for part, condition in parts if condition]
 
+    def update_msg_content_header(self) -> bool:
+        """
+        Update the content header of a message box (if required).
+        """
+        update_required = False
+        content_header = None
```

Comment:
This isn't the content_header, but the author field? If so, let's name it that? Maybe set `author_column = 1` and use that in the indexing to make it clearer?